### ISSUE TITLE
Error on first header column name

### ISSUE THREAD

**Issue by haroldcb** [abdennour/react-csv#15]
First of all, thank you for your work!
However, I got an issue using your lib : when I export data, I got a comma before first header element...
`const dataTest = [{name:"xxxxxx",firstName:"xxxxxx",email:"xxxxxx@xxxxxx.com"},{name:"xxxxxx",firstName:"xxxxxx",email:"xxxxxx@xxxxxx.com"}];` 

In the exported file, I have :  

> ,"name"

 as column name.

Using the link with : 
`<CSVLink data={dataTest} filename={"export.csv"} className="btn btn-primary" target="_blank" separator={";"}>
export
</CSVLink>`

If you could help me with it it would be great!
Thanks in advance

**Comment by jyapayne** [comment]
Looks like the problem lies [here](https://github.com/abdennour/react-csv/blob/master/src/core.js#L44):

```js
`data:text/csv;charset=utf-8,\uFEFF,${toCSV(data, headers, separator)}`
```

should be

```js
`data:text/csv;charset=utf-8,\uFEFF${toCSV(data, headers, separator)}`
```

Notice the missing comma after `\uFEFF`

**Comment by dangnelson** [comment]
Also seeing this. Can we get a quick fix for this?

**Comment by phwareboss** [comment]
This is a problem for me as well

**Comment by raza30492** [comment]
I am also getting the same problem. please fix asap.

**Comment by abdennour** [comment]
Sorry guys! I was in trip the last days. this issue is now fixed with  the NPM release  `1.0.8` .
